libs/sdk-core/src/persist/db.rs

```diff
@@ -190,6 +190,14 @@ impl SqliteStorage {
              
              DROP TABLE old_swaps;            
             "),
+            M::up(
+                "
+             CREATE TABLE IF NOT EXISTS lnurl_success_action (
```

Comment:
I think it is better to cal this table in a more generic way such as `payments_external_info` and create a foreign key from the Payments.id column to a column here called payment_id (for lightning payments it is the hash).
Also I think we should call the value column `lnurl_success_action`.

This way we gain the following advantages:
1. It is clear that it is associated with the payments table and the db validates that using the foreign key.
2. We for sure will need to map more data in the future (such as lnurl metadata) so we can do that by simply adding more columns to this table.

Reply:
Done